pocket_world_application/src/main/java/org/exschool/pocketworld/model/Time.java

```diff
@@ -0,0 +1,38 @@
+package org.exschool.pocketworld.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Time {
```

Comment:
you should use embedded id here http://stackoverflow.com/questions/3585034/how-to-map-a-composite-key-with-hibernate